net/zerotier/patches/0003-add-cerrno-header-for-str-errno.patch

```diff
@@ -1,7 +1,7 @@
-From e06177a74f1e4314baf17c52f360dabcb78e69cd Mon Sep 17 00:00:00 2001
+From 7d6797058f3d95e02c76d2fe7c665b5cbbcf4b29 Mon Sep 17 00:00:00 2001
 From: Moritz Warning <moritzwarning@web.de>
 Date: Sun, 29 Nov 2020 19:45:36 +0100
-Subject: [PATCH 7/8] add cerrno header for (str)errno
+Subject: [PATCH 3/5] add cerrno header for (str)errno
```

Comment:
Can we send this patch to upstream as well?

Reply:
This patch was already in when we asked them (ZT) to see what they can upstream. So I assume that they do not want it or overlooked it. Then they can look at it next time - imho.

Reply:
This is already merged as I can see here:
https://github.com/zerotier/ZeroTierOne/pull/1110

It should be included in 1.8.6 release. So this could be dropped, right?

Reply:
@adamierymenko please have a look.

Reply:
@BKPepe true, I've removed it now.